### ISSUE TITLE
ISISICP: move to Apps

### ISSUE THREAD

**Issue by FreddieAkeroyd** [ISISComputingGroup/IBEX#7624]
To resolve firewall issue on SXD:

* Move c:\labview modules\dae to c:\instrument\apps\ISISICP
* create junction from c:\labview modules\dae to c:\instrument\apps\ISISICP
* add firewall rules for A:\ISISICP\service\x64\release\isisicp.exe, A:\ISISICP\service\x64\release\isisdatasvr.exe and A:\ISISICP\service\x64\release\qxtalk.exe

This should be roled out more widely, though SXD is a special case in needing it now

**Comment by GRyall** [comment]
@FreddieAkeroyd Is this still current? It is on the task list as things stand.

**Comment by FreddieAkeroyd** [comment]
Yes, it should still be done - i'll make a script and we can probably do as part of deploy

**Comment by FreddieAkeroyd** [comment]
currently deployed to zoom, wish, sans2d, polref, offspec, larmor, inter, sxd, loq, hifi, chronus , argus
